doc/source/cluster/autoscaling.rst

```diff
@@ -36,6 +36,17 @@ The basic autoscaling config settings are as follows:
     # considered idle if there are no tasks or actors running on it.
     idle_timeout_minutes: 5
 
+
+Manually Adding Nodes
+---------------------
+
+Nodes can be manually added to an autoscaled cluster. To do so, the `ray-cluster-name` tag should be set and `ray-node-type` tag should be set to `unmanaged`.
```

Comment:
Can you comment as to why you might want to do this? I.e., connect an application to the cluster with num_cpus=0?